solr/core/src/java/org/apache/solr/client/solrj/embedded/JettySolrRunner.java

```diff
@@ -382,7 +387,8 @@ public void lifeCycleStarted(LifeCycle arg0) {
 
         root.getServletContext().setAttribute(SolrDispatchFilter.PROPERTIES_ATTRIBUTE, nodeProperties);
         root.getServletContext().setAttribute(SolrDispatchFilter.SOLRHOME_ATTRIBUTE, solrHome);
-
+        coreContainerProvider = new CoreContainerProvider();
+        coreContainerProvider.init(root.getServletContext());
```

Comment:
This appears to be an issue in the design of CalciteSolrDriver, not really a function of this change. The INSTANCE or the cache snould instead be a per core container map. This could be a root of flakiness in tests? If more than one Core container is started per JVM/classloader then the cache gets replaced by the subsequent CoreContainer.load() call.

Reply:
SOLR-15775

Reply:
ignore